docs/source/perf/spmd_basic.md

```diff
@@ -61,56 +65,133 @@ SPMD with other distributed libraries.
 
 ### Mesh
 
-For a given cluster of devices, a physical mesh is a representation of
-the interconnect topology.
+The SPMD programming model is built around the concept of a device mesh, commonly
+referred to as a mesh. A device mesh is a logical N-dimensional arrangement of
+compute devices (e.g. TPU cores) where the axes are used as shorthands to
+define sharded computation. It is logical in the sense that the mesh shape does
+not necessarily have to reflect the physical network layout. You could create
+different kinds of device meshes over the same set of physical devices. For
+example, a 512-core TPU slice could be treated as a 3D mesh of 16×16×2, a 2D mesh
+of 32×16, or a 1D mesh of 512, depending on how we want to partition tensors. You
+may use the `Mesh` class to create a device mesh.
```

Comment:
You use the `Mesh` class to create a device mesh.

Reply:
done